creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -150,13 +152,19 @@ const handleSyncFromPrimary = async ({
      * Secondary requests export of new data by passing its current max clock value in the request.
      * Primary builds an export object of all data beginning from the next clock value.
      */
+    const startFetchExportDate = Date.now()
     const { fetchedCNodeUser, error, abort } = await fetchExportFromNode({
       nodeEndpointToFetchFrom: creatorNodeEndpoint,
       wallet,
       clockRangeMin: localMaxClockVal + 1,
       selfEndpoint: thisContentNodeEndpoint,
       logger
     })
+    const fetchExportDurationMs = Date.now() - startFetchExportDate
```

Comment:
👀 if its just logging we want, we can use this fn. we can also modify this fn to take in a key for `duration`, instead of the default duration key https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/logging.js#L130

Reply:
/ we might not need to create a child logger each time if the purpose is just to log fields with a key

Reply:
changed!